packages/faustwp-core/src/hooks/useLogout.tsx

```diff
@@ -25,8 +25,17 @@ export function useLogout() {
       return;
     }
 
+    const isPreview = window?.location?.search?.includes('preview=true');
+
     if (redirectUrl) {
       window.location.assign(redirectUrl);
+    } else if (isPreview) {
+      const publicUrlPath = process.env.NEXT_PUBLIC_URL;
```

Comment:
I think we should remove the check for `NEXT_PUBLIC_URL`. This is not a valid env var in Faust, only in experimental app router, and I think we can remove that soon.